.rspec

```diff
@@ -1,2 +1 @@
---colour
--r turnip
+--colour -r turnip/rspec
```

Comment:
How we load Turnip has changed; this is necessary to get the features running inside RSpec / on Travis.